### PR TITLE
qa/cephadm: centos_8.3_container_tools_3.0.yaml

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Let's avoid latest stable

Signed-off-by: Sebastian Wagner <sewagner@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: qa/distros/podman/centos_8.3_container_tools_3.0.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  selinux:
+    whitelist:
+      - scontext=system_u:system_r:logrotate_t:s0
+
+tasks:
+- pexec:
+    all:
+    - sudo cp /etc/containers/registries.conf /etc/containers/registries.conf.backup
+    - sudo dnf -y  module reset container-tools
+    - sudo dnf -y  module install container-tools:3.0
```
Comment (fmount) [inline qa/distros/podman/centos_8.3_container_tools_3.0.yaml:13]: s/install/enable

Comment (sebastian-philipp) [reply qa/distros/podman/centos_8.3_container_tools_3.0.yaml:13]: https://docs.fedoraproject.org/en-US/modularity/using-modules-switching-streams/ ?

Comment (fmount) [reply qa/distros/podman/centos_8.3_container_tools_3.0.yaml:13]: ack

**Comment by yuriw** [comment]
`teuthology-suite -v --ceph-repo https://github.com/ceph/ceph-ci.git --suite-repo https://github.com/sebastian-philipp/ceph.git -c master -m smithi -s rados/cephadm -k distro -p 71 --suite-branch centos_8.3_container_tools_3.0.yaml -S f1cafa5cbec2e5708a40c39854ba202aa58a7e4c --force-priority --subset 111/120000`

https://pulpito.ceph.com/yuriw-2021-08-19_17:00:30-rados:cephadm-master-distro-basic-smithi/

**Review by jdurgin — APPROVED** [review]

**Comment by sebastian-philipp** [comment]
follow-up #42863

**Pull request merged**